lib/vagrant-service-manager/command.rb

```diff
@@ -217,6 +228,28 @@ def print_docker_env_info(guest_ip, port, secrets_path, machine_uuid)
           @env.ui.info(puts(message))
         end
       end
+
+      def print_vagrant_box_version
+        # Prints the version of the vagrant box, parses /etc/os-release for version
+        @@OS_RELEASE = "/etc/os-release"
```

Comment:
Why as Global variable?

As this is constant, `OS_RELEASE` would be fine. 

However, we should have a separate file as module to store all constants. Let's keep that as part of refactoring #49 .